pages/Food.md

```diff
@@ -0,0 +1,18 @@
+The Food category contains the following foods and juices:
+* [Fortune Cookie](Fortune-Cookie.md)
+* [Diet Cookie](Diet-Cookie.md)
+* [Beef Jerky](Meat-Jerky.md)
+* [Pork Jerky](Meat-Jerky.md)
+* [Chicken Jerky](Meat-Jerky.md)
+* [Mutton Jerky](Meat-Jerky.md)
+* [Fish Jerky](Meat-Jerky.md)
+* [Magic Sugar](Magic-Sugar.md)
+* [Monster Jerky](Monster-Jerky.md)
+* [Apple Juice](Juice.md)
+* [Carrot Juice](Juice.md)
+* [Melon Juice](Juice.md)
```

Comment:
I'd maybe link to the specific header in this file. https://link/file#Header

Reply:
Thank you, I hadn't thought of it. The page doesn't have any headers yet but I will keep this in mind for the future.